cmd/csi-addons/encryptionkeyrotation.go

```diff
@@ -0,0 +1,62 @@
+/*
+Copyright 2024 The Kubernetes-CSI-Addons Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/csi-addons/kubernetes-csi-addons/internal/proto"
+	"github.com/csi-addons/kubernetes-csi-addons/internal/sidecar/service"
+)
+
+type EncryptionKeyRotation struct {
+	grpcClient
+
+	persistentVolume string
+}
+
+var _ = registerOperation("EncryptionKeyRotation", &EncryptionKeyRotation{})
+
+func (ekr *EncryptionKeyRotation) Init(c *command) error {
+	ekr.persistentVolume = c.persistentVolume
+
+	if ekr.persistentVolume == "" {
+		return fmt.Errorf("persistent volume name is not set")
```

Comment:
`persistent volume` to `PersistentVolume` use errors.New()

Reply:
Can we use capitalized error statements?

Reply:
lets try and see if CI complains, if not we can have it as it is

Reply:
It is a golang convention to have error strings start with lowercase and without period. I do not think we have checks in CI for this though.

Reply:
we can add error strings with upper case, it just shows some indications in code base, but good to maintain the convention as used elsewhere in the project.
So, in my opinion it's fine to use `PersistentVolume`